src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -559,6 +559,10 @@ public void GetNameAndArityOfSimpleName(SyntaxNode node, out string name, out in
             }
         }
 
+        public bool LooksGeneric(SyntaxNode simpleName)
+            => simpleName.IsKind(SyntaxKind.GenericName) ||
+               simpleName.GetLastToken().GetNextToken().Kind() == SyntaxKind.LessThanToken;
```

Comment:
❓ For the sake of this feature, why would we not want to restrict this behavior to only occurring when the `<` token is on the same line as the identifier?